cardano-api/src/Cardano/Api/Value.hs

```diff
@@ -1,6 +1,8 @@
+{-# LANGUAGE BlockArguments #-}
```

Comment:
Let's remove this pragma. I don't think its work it for the sake of not specifying parentheses in your lamba function.

Reply:
done